docs/source/trainer/algorithms.rst

```diff
@@ -3,11 +3,8 @@
 
 Under construction |:construction:|
 
-Included in the Composer library is a suite of algorithmic speedup algorithms.
-These modify the basic training procedure, and are intended to be *composed*
-together to easily create a complex and hopefully more efficient training routine.
-While other libraries may have implementations of some of these, the implementations
-in Composer are specifically written to be combined with other methods.
+Composer has an awesome library of speedups that can be composed together
+to easily create efficient training recipes.
```

Comment:
Hesitant to call our own library "awesome." Could probably call papers or methods "awesome" if we did it carefully.

The word "recipes" sounds normal to us but I don't know what it means when I put on my data scientist hat. Do I need training "recipes"? Is a "recipe" some abstraction in this library? I can kind of guess what it means but I'm not sure unless it's defined.

```suggestion
Composer has a collection of speedup methods that can be composed together
to easily create efficient training pipelines.
```

Reply:
^ although if "pipelines" suggests something more comprehensive to others (e.g., whole ETL pipeline), then "recipes" might be the least confusing phrasing. Or just "logic" or "scripts" or "programs". These might be safest.